e2e/cypress/integration/enterprise/system_console/team_and_channel_asign_roles_spec.js

```diff
@@ -81,6 +81,82 @@ describe('System Console', () => {
 
         // * Check to make the the current role text is displayed as Member
         cy.findByTestId('current-role').should('have.text', 'Member');
+
+        // # Remove all exisiting groups
+        cy.get('#groups-list--body').then((el) => {
+            if (el[0].childNodes[0].innerText !== 'No groups specified yet') {
+                for (let i = 0; i < el[0].childNodes.length; i++) {
+                    cy.get('#group-actions').click();
+                }
+
+                // # Save the setting
+                cy.get('#saveSetting').click();
+            }
+        });
+
+        // # Wait to ensure it has saved before reloading
+        cy.wait(500); //eslint-disable-line cypress/no-unnecessary-waiting
+
+        // # Reload to ensure it's been saved
+        cy.reload();
+
+        // * Assert that the group was removed successfully
+        cy.get('#groups-list--body').then((el) => {
+            expect(el[0].childNodes[0].innerText).equal('No groups specified yet');
+        });
+    });
+
+    it('MM-21789 - Add a group and change the role and then save and ensure the role was updated on team configuration page', () => {
+        const teamName = 'eligendi';
+        cy.apiLogin('sysadmin');
+
+        // # Go to system admin page and to team configuration page of channel "eligendi"
+        cy.visit('/admin_console/user_management/teams');
+
+        // # Search for the team.
+        cy.findByTestId('search-input').type(`${teamName}{enter}`);
+        cy.findByTestId(`${teamName}edit`).click();
+
+        // # Wait until the groups retrieved and show up
+        cy.wait(500); //eslint-disable-line cypress/no-unnecessary-waiting
+
+        // # Remove all exisiting groups
+        cy.get('#groups-list--body').then((el) => {
+            if (el[0].childNodes[0].innerText !== 'No groups specified yet') {
+                for (let i = 0; i < el[0].childNodes.length; i++) {
+                    cy.get('#group-actions').click();
+                }
+
+                // # Save the setting
+                cy.get('#saveSetting').click();
+            }
+        });
+
+        // # Add the first group in the group list then save
+        cy.findByTestId('add-group').click();
+        cy.get('#multiSelectList').first().click();
+        cy.get('#saveItems').click();
+
+        // * Ensure default role is Member
+        cy.findByTestId('current-role').should('have.text', 'Member').click();
+
+        // * Assert that only one option exists in the dropdown for changing roles
+        cy.get('#role-to-be-menu').then((el) => {
+            expect(el[0].firstElementChild.children.length).equal(1);
+        });
+
+        // # Continue changing the role to Team Admin
+        cy.get('#role-to-be').click();
```

Comment:
Hi @hahmadia 
At times, when the Cypress test clicks on the dropdown to change the role, the Save button does not gets enabled, even if the role dropdown was selected. It looks like the trigger action when the role is changed is not properly triggered when we click through Cypress test. Can you please check and update it throughout the test.